src/internal/connector/graph_connector.go

```diff
@@ -49,9 +49,17 @@ type GraphConnector struct {
 	itemClient *http.Client // configured to handle large item downloads
 
 	tenant      string
-	Sites       map[string]string // webURL -> siteID and siteID -> webURL
 	credentials account.M365Config
 
+	// TODO: remove in favor of the maps below.
+	Sites map[string]string // webURL -> siteID and siteID -> webURL
+
+	// maps of resource owner ids to names, and names to ids.
+	// not guaranteed to be populated, only here as a post-population
+	// reference for processes that choose to populate the values.
+	ResourceOwnerIDToName map[string]string
+	ResourceOwnerNameToID map[string]string
```

Comment:
do these need to be exported? I'm hesitant to say GC should guarantee that some data is pre-populated as that puts us in an awkward situation when it comes to repeating work etc. Having these public makes it harder to remove those guarantees because there's no way to hide changes in functionality (e.x. no function to change the implementation of behind the scenes)